BridgeAppSDK/SBATrackedDataResult.swift

```diff
@@ -37,6 +37,10 @@ public class SBATrackedDataSelectionResult: ORKQuestionResult {
     
     public var selectedItems: [SBATrackedDataObject]?
     
+    override init() {
```

Comment:
Yes, with swift, any init that you do not override is no longer a valid initializer (throws) and therefore, when copyWithZone instantiates using [[[self class] alloc] init] you get a runtime exception.